sunpy/conftest.py

```diff
@@ -33,6 +34,12 @@
 console_logger.setLevel('INFO')
 
 
+@pytest.fixture(scope='session', autouse=True)
+def jsoc_test_email():
+    # JSOC Test email
+    pytest.jsoc_test_email = "nabil.freij@gmail.com"
```

Comment:
Is this really a recommended pattern, to just attach an attribute to an imported module? This seems real gross.

Reply:
I'll change this in a PR

Reply:
It was more fun this way.